<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chageattr</class>
 <widget class="QWidget" name="chageattr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>id:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ROAD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CHARGE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IMPORT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EXPORT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SHELF</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
